aspnetcore/blazor/javascript-interop.md

```diff
@@ -243,3 +243,23 @@ The class library handles embedding JavaScript resources in the built assembly.
 The built NuGet package is referenced in the app's project file the same way that any NuGet package is referenced. After the package is restored, app code can call into JavaScript as if it were C#.
 
 For more information, see <xref:blazor/class-libraries>.
+
+## Harden JS interop calls
+
+JS interop may fail due to networking errors and should be treated as unreliable. By default, a Blazor Server app times out JS interop calls on the server after one minute. If an app can tolerate a more aggressive timeout, such as 10 seconds, set the timeout using one of the following approaches:
+
+* Globally in `Startup.ConfigureServices`, specify the number of `{SECONDS}` for the timeout:
```

Comment:
Why "Seconds"?  Why not just say 

>  Globally in `Startup.ConfigureServices`, specify the timeout duration:

Reply:
I did that because its a placeholder. I'll use your version on the next commit.